src/web/components/ArticleTitle.tsx

```diff
@@ -50,14 +51,18 @@ export const ArticleTitle = ({
     sectionUrl,
     guardianBaseURL,
     pillar,
-    badge,
     inLeftCol,
     fallbackToSection = true,
+    badge,
+    frontendAssetsFullURL,
 }: Props) => (
     <div className={cx(inLeftCol && sectionStyles, badge && badgeContainer)}>
         {badge && (
             <div className={titleBadgeWrapper}>
-                <Badge svgSrc={badge.svgSrc} linkTo={badge.linkTo} />
+                <Badge
+                    imageUrl={`${frontendAssetsFullURL}${badge.imageUrl}`}
```

Comment:
There's a lib for joining urls (urlJoin) to guard against the double slash thing

Reply:
not needed as moved this up to Frontend!